include/dali/c_api.h

```diff
@@ -137,6 +138,17 @@ enum {
    * Only relevant when the input is either pinned host memory or device memory
    */
   DALI_use_copy_kernel = (1 << 2),
+
+  /**
+   * Override the `no_copy` specified for given External Source and force the data to be copied.
+   */
+  DALI_ext_force_copy = (1 << 3),
+
+  /**
+   * Override the `no_copy` specified for given External Source and pass the data directly to the
+   * Pipeline.
+   */
+  DALI_ext_force_no_copy = (1 << 4),
```

Comment:
What lacks for me in this docs is information, how does it different from specifying `no_copy` in "regular" way (by "regular" I mean whatever that must be overwritten)